release_notes_generator/model/pull_request_record.py

```
@@ -0,0 +1,312 @@
+        Returns:
+            list[str]: A sorted list of GitHub usernames of contributors, excluding the main author.
+        """
+        if not self._pull_request:
+            return []
+
+        return []
+
+        # TODO - contributors have to be managed in future, make a research if get_commit can return some
+        #  new information in compare to mined ones
+        # Get unique contributors from commits
+        contributors = set()
```

Ruff (0.12.2)

Local variable `contributors` is assigned to but never used

Remove assignment to unused variable `contributors`

(F841)